### ISSUE TITLE
Support for JMX notifications

### ISSUE THREAD

**Issue by johntdyer** [jmesnil/jmx4r#9]
Does JMX4R support RMI notifications ?  Is it does do you maybe have an example ?

**Comment by jmesnil** [comment]
No, there is currently no support to register notification listeners. Would you be interested to contribute a patch to add this feature?
